api/src/certification/evaluation/domain/services/scoring/score-complementary-certification-v2.js

```diff
@@ -77,12 +86,16 @@ export async function scoreComplementaryCertificationV2({
 
 async function _getComplementaryCertificationResultInformation({
   complementaryCertificationScoring,
-  hasComplementaryReferential,
+  complementaryCertificationKey,
   complementaryCertificationBadgesRepository,
   assessmentResult,
   minimumReproducibilityRateLowerLevel,
 }) {
-  if (hasComplementaryReferential && assessmentResult.isValidated()) {
+  if (
+    complementaryCertificationKey &&
+    complementaryCertificationKey !== ComplementaryCertificationKeys.CLEA &&
+    assessmentResult.isValidated()
```

Comment:
```suggestion
    complementaryCertificationKey && assessmentResult.isValidated()
```

La clé ne peut ici pas être autre chose que CORE ou undefined avec la vérification faite au dessus.

Je serais personnellement favorable à ne pas remplacer la valeur CLEA par undefined à la ligne 70 et garder ce check ici.